sprint mission 5/src/services/articleService.ts

```diff
@@ -0,0 +1,67 @@
+import prisma from '../db.js';
+
+type CreateArticleInput = {
```

Comment:
input 들만 모아서 dto 를 구성하고 가져다가 사용만 하시면 됩니다.

```suggestion
class CreateArticleInputDto {
    ...
}
```